src/runtime/contrib/tensorrt/tensorrt_builder.cc

```diff
@@ -40,30 +40,30 @@ namespace contrib {
 TensorRTBuilder::TensorRTBuilder(TensorRTLogger* logger,
                                  const std::vector<const DLTensor*>& data_entry,
                                  size_t max_workspace_size, bool use_implicit_batch, bool use_fp16,
-                                 int batch_size)
+                                 int batch_size, nvinfer1::IInt8Calibrator* calibrator)
     : data_entry_(data_entry),
       max_workspace_size_(max_workspace_size),
       use_implicit_batch_(use_implicit_batch),
       use_fp16_(use_fp16),
       batch_size_(batch_size) {
   // Create TRT builder and network.
   builder_ = nvinfer1::createInferBuilder(*logger);
-#if TRT_VERSION_GE(6, 0, 1)
```

Comment:
Why does these version macros need to be removed?

In my experience in higher version of TRT, use api like `builder_->setInt8Mode` will throw out some warning(Even in TRT8, some api has been deprecated ... Better to test these in a higher version.)

Reply:
> Why does these version macros need to be removed?
> 
> In my experience in higher version of TRT, use api like `builder_->setInt8Mode` will throw out some warning(Even in TRT8, some api has been deprecated ... Better to test these in a higher version.)

I have re-added them into the code base